scripts/run_local_network.sh

```diff
@@ -257,27 +298,31 @@ if $transactions; then
   folder=$nodesfolder/fish_1
   keyfile=$ledgerfolder/online_fish_keys/online_fish_account_1
   pubkey=$(cat $ledgerfolder/online_fish_keys/online_fish_account_1.pub)
-  port=5006
-  transaction_frequency=5
+  rest_server="http://127.0.0.1:5001/graphql"
```

Comment:
Should all the shell variables be upper-cased? It's a bit confusing to have a mix of cases ...

Reply:
OK, I wasn't aware of that convention.